### PR TITLE
Added Checklist highlighting for a lot of things

### PR DESCRIPTION
**Pull request by DoToAdventures**
**Summary of Changes**

I added checklist hightlighting of instruments and knobs for a lot checklist items by modifying modelbehaviourdefs and model files, adding PART_IDs to things that did not have them assigned yet.

### CODE REVIEW

File: A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/model/A320_NEO_INTERIOR.xml [hunk 1]
```diff
@@ -2448,13 +2465,14 @@
 		<UseTemplate Name="ASOBO_GT_Interaction_Tooltip">
 			<NODE_ID>KNOB_CHRONO_RUN</NODE_ID>
 		</UseTemplate>
-
+	<CameraTitle>ECAM</CameraTitle>
```
Comment (Benjozork) [inline A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/model/A320_NEO_INTERIOR.xml:2468]: Indentation here ;)

**Comment by DoToAdventures** [comment]
aaaaaaahhh don't fix Indentation that much lol

File: A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/model/A320_NEO_INTERIOR.xml [hunk 2]
```diff
@@ -2592,6 +2614,7 @@
 				<NODE_ID>PUSH_OVHD_CALLS_AFT</NODE_ID>
 			</UseTemplate>
 		</Component>
+			<CameraTitle>Overhead</CameraTitle>
```
Comment (Benjozork) [inline A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/model/A320_NEO_INTERIOR.xml]: And here ;)

**Comment by Benjozork** [comment]
Ahahaha only two don't worry !!

Thank you for those changes, I never thought this would even work !

**Comment by DoToAdventures** [comment]
I just alt+shift+f'd it, it indented some things that I haven't touched as well, I guess it's fine though?

**Comment by Benjozork** [comment]
> I just alt+shift+f'd it, it indented some things that I haven't touched as well, I guess it's fine though?

Yes perfect indentation everywhere ! Good ! 🤣

**Comment by AdenFlorian** [comment]
"ECAM Pages: ..." step has wrong camera, there's another camera that shows the screens and the buttons:

current camera:
image

better camera:
image

**Comment by AdenFlorian** [comment]
SEAT BELTS doesn't have an auto setting, jsut ON and OFF:
image

**Comment by AdenFlorian** [comment]
this is using the wrong camera:
image

**Comment by AdenFlorian** [comment]
wrong camera:
image

**Comment by AdenFlorian** [comment]
wrong parts, step says PFD and ND brightness but it highlights the FCU brightness knobs:
image

**Comment by AdenFlorian** [comment]
wrong camera:
image

**Comment by AdenFlorian** [comment]
wrong camera:
image

**Comment by AdenFlorian** [comment]
this step says APU MASTER but APU GEN is highlighted:
image

**Comment by AdenFlorian** [comment]
Very nice overall though!

**Review by AdenFlorian — APPROVED** [review]
tested and looks pretty good! I left comments on things that could be improved, but up to you if you want to merge now or not

**Comment by DoToAdventures** [comment]
Fixed everything, should be good now!

File: A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/Checklist/Library.xml [hunk 3]
```diff
@@ -107,7 +107,7 @@
          <CheckpointDesc SubjectTT="TT:Exterior Lights" ExpectationTT="TT:As required" />
       </Checkpoint>
       <Checkpoint Id="A320_CPREP1_NO_SMOKING_SEATBELTS">
-         <CheckpointDesc SubjectTT="TT:No Smoking and Seatbelt Signs" ExpectationTT="TT:AUTO" />
+         <CheckpointDesc SubjectTT="TT:No Smoking and Seatbelt Signs" ExpectationTT="TT:ON" />
```
Comment (AdenFlorian) [inline A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/Checklist/Library.xml:110]: So, its tricky, seat belts only have on and off, and should be turned on in this step, but no smoking does have an auto, and it should be auto in this step, not on (i believe)

Comment (DoToAdventures) [reply A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/Checklist/Library.xml:110]: I will seperate the two

Comment (Venorcis) [reply A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/Checklist/Library.xml]: I think you reversed the ON and AUTO eventually now @DoToAdventures, I swapped it in my latest pull (https://github.com/flybywiresim/a32nx/pull/563)

**Comment by AdenFlorian** [comment]
Thanks!

**Pull request merged**